creator-node/src/config.js

```diff
@@ -453,6 +453,30 @@ const config = convict({
     env: 'snapbackDevModeEnabled',
     default: false
   },
+  snapbackReconfigEnabled: {
+    doc: 'Enables replica set reconfiguration or not. Currently enabled for dev mode, disabled for prod.',
```

Comment:
there needs to be a corresponding value for this in the base.env file to set this to true for dev mode right?

Reply:
same for `snapbackModuloBase` below

Reply:
yes, will add to this PR values in base.env for local setup